src/tr.i18n.js

```diff
@@ -84,29 +82,27 @@ module.exports = {
     'Forward': 'İleri',
     'Go to Previous Fork': 'Önceki Çatala Git',
     'Go to Next Fork': 'Sonraki Çatala Git',
-    'Go to Previous Comment': 'Önceki Yoruma Git',
-    'Go to Next Comment': 'Sonraki Yoruma Git',
+    'Go to Previous Comment': 'Önceki Açıklamaya Git',
+    'Go to Next Comment': 'Sonraki Açıklamaya Git',
     'Go to Beginning': 'Başlangıca Git',
     'Go to End': 'Sona Git',
     'Go to Main Variation': 'Ana Varyasyona Git',
     'Go to Previous Variation': 'Önceki Varyasyona Git',
     'Go to Next Variation': 'Sonraki Varyasyona Git',
     'Go to Move Number': 'Hamle Numarasına Git',
-    'Enter a move number to go to': 'Gitmek için bir hamle numarası girin',
+    'Enter a move number to go to': 'Gidilecek hamle numarasını girin',
     'Go to Next Game': 'Sonraki Oyuna Git',
     'Go to Previous Game': 'Önceki Oyuna Git'
   },
   'menu.engines': {
     'Engines': 'Motorlar',
     'Show Engines Sidebar': 'Motor Kenar Çubuğunu Göster',
-    'Toggle Analysis': 'Analiz Geçişi',
-    'None of the attached engines support analysis.':
-      'İlişkilendirilmiş motorlardan hiç biri analizi desteklemiyor.',
-    'Start Engine vs. Engine Game': 'Motor vs. Motor Oyunu Başlat',
-    'Stop Engine vs. Engine Game': 'Motor vs. Motor Oyununu Durdur',
+    'Toggle Analysis': 'Analizi Aç/Kapat',
```

Comment:
After spending more time by using program i have realized that "Analizi Başlat/Durdur" is more proper translation.